src/components/layout/HeaderMenu.vue

```diff
@@ -15,7 +15,7 @@ export default {
         { label: this.$t("menu.contacts"), route: "/profiles/contacts" },
         {
           label: this.$t("menu.trailsTreasuries"),
-          route: "/trails/treasuries",
+          route: "/",
```

Comment:
We want to redirect to `/trails/ballots` instead, to show the proposals, not the groups. 

Also, I think the best approach here would be to make route `/ ` redirect to `/trails/ballots` to minimize the chances of people accessing the link and reaching to an empty page.